cloud-aws/src/test/java/com/sequenceiq/cloudbreak/cloud/aws/view/AwsRdsDbParameterGroupViewTest.java

```diff
@@ -111,6 +110,7 @@ void getDBParameterGroupFamilyTestWhenPgSqlAndUnsupportedMajorVersion(String tes
     static Object[][] validVersionDataProvider() {
         return new Object[][]{
                 // testCaseName version familyExpected
+                {"version is null", null, "postgres10"},
                 {"version 9.5", "9.5", "postgres9.5"},
                 {"version 9.6", "9.6", "postgres9.6"},
                 {"version 10.1", "10.1", "postgres10"},
```

Comment:
Perhaps your intention was to replace the `"postgres10"` occurrences with `POSTGRES_10`?